src/System.Runtime.Loader/ref/System.Runtime.Loader.csproj

```diff
@@ -4,8 +4,10 @@
   <PropertyGroup>
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="System.Reflection.Metadata.cs" />
```

Comment:
It is actually very common that the library contains namespaces that are different from its name...